common/predictive-text/README.md

```diff
@@ -0,0 +1,404 @@
+Keyman LMLayer prototype
+========================
+
+[![Build Status](https://travis-ci.org/eddieantonio/keyman-lmlayer-prototype.svg?branch=master)](https://travis-ci.org/eddieantonio/keyman-lmlayer-prototype)
```

Comment:
We'll obviously need to get this changed over in the final version, though we're not in the place to do that quite yet.  Just making a note for future reference.